frontend/vue/components/UserAccount/UserAccountLayout.vue

```diff
@@ -46,6 +46,9 @@
         <UserProgress />
       </div>
       <div v-if="activeSection === sectionList[1].hash">
+        <ClassroomSection />
+      </div>
+      <div v-if="activeSection === sectionList[2].hash">
```

Comment:
This v-if conditionals could be changed to [v-else-if](https://es.vuejs.org/v2/guide/conditional.html#v-else-if) i think.

Reply:
yes, this worked nicely 👍🏽